app/views/content/train-to-be-a-teacher/how-to-choose-your-teacher-training.md

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
@philspray I'm wondering whether the URL need changing to `/how-to-choose-your-teacher-training-course`

(ie adding `-course` on the end)

will this make much of a difference?

Reply:
I don't think it will make much difference. It is easy to add on the end.